package.json

```diff
@@ -17,9 +17,11 @@
     "format:md": "prettier --write --parser markdown '**/*.md'",
     "format:yaml": "prettier --write --parser yaml '**/*.yaml'",
     "start": "yarn playground:start",
-    "release": "npm run build; lerna publish --exact --dist-tag next",
-    "release-canary": "npm run build; lerna publish --exact --dist-tag canary --canary --preid canary --yes",
-    "release-to-latest": "lerna exec --no-bail --no-private --no-sort --stream -- '[ -n \"$(npm v . dist-tags.next)\" ] && npm dist-tag add ${LERNA_PACKAGE_NAME}@$(npm v . dist-tags.next) latest'",
+    "prerelease": "npm run build",
+    "release": "lerna publish --exact --dist-tag next",
+    "prerelease:canary": "npm run build",
```

Comment:
The `:` separation is better in the long term cause scripts like `run-all` can use it.